app/views/equipment_items/show.html.erb

```diff
@@ -21,11 +21,25 @@
 
 <%# NOTES %>
 <div class="row">
-  <div class="col-md-12">
-    <% unless @equipment_item.notes.blank? %>
-      <hr>
-      <h3>Notes</h3>
-      <div class='notes-container'><%= markdown(@equipment_item.notes) %></div>
+  <% unless @equipment_item.notes.blank? %>
```

Comment:
If we're leaving the form there no matter what, we should just have a blank notes section (maybe with a message "No notes yet!" or something if there are no notes) so that the form doesn't magically jump to the other side of the page. Also, there should be a single horizontal rule above the entire section, not separate ones above each side.